rfc6962-bis.xml

```diff
@@ -1130,6 +1130,43 @@ entry specified by <spanx style="verb">start</spanx>.
         <t>See <xref target="verify_hash" /> for an outline of how to use a complete list of <spanx style="verb">leaf_input</spanx> entries to verify the <spanx style="verb">root_hash</spanx>.</t>
       </section>
 
+      <section title="Get Entry Number for SCT">
+	<t>
+	  GET https://&lt;log server&gt;/ct/v2/get-entry-for-sct
```

Comment:
As discussed offline, let's make this an optional API. 
My argument for not making it mandatory is that while it would ease lookup in some cases, it is not an essential part of the protocol (i.e. the same functionality can be obtained by downloading the entire log) and adds a burden on log implementation.